src/App.jsx

```diff
@@ -1,10 +1,115 @@
+import {useState} from 'react';
+import axios from 'axios';
+import Header from "./components/Header/Header.jsx";
+import Button from "./components/Button/Button.jsx";
+import Footer from "./components/Footer/Footer.jsx";
+import InputField from "./components/InputField/InputField.jsx";
+import getRegionClassName from "./helpers/getRegionClassName.js";
+import spinningGlobe from './assets/spinning-globe.gif';
+import map from './assets/world_map.png';
 import './App.css';
+import formatNumberToMillions from "./helpers/formatNumberToMillions.js";
+
 
 function App() {
+    const [countries, setCountries] = useState([]);
+    const [error, setError] = useState('');
+    const [countryInformation, setCountryInformation] = useState('');
```

Comment:
`countryInformation` zal een object worden en daardoor vind ik een lege string geen valide default waarde. Ik zou zelf kiezen voor `undefined`
```suggestion
    const [countryInformation, setCountryInformation] = useState();
```